bundler/src/main/resources/log4j2.yml

```diff
@@ -18,10 +18,27 @@ Configuration:
       follow: true
       PatternLayout:
         pattern: "${LOG_PATTERN}"
+    RollingFile:
+      name: File
+      fileName: ${logDir}/bundler.log
+      filePattern: "${logDir}/bundler.log.%d{yyyy-MM-dd-hh}"
+      Policies:
+        TimeBasedTriggeringPolicy:
+          interval: 1
+          modulate: true
+      DefaultRolloverStrategy:
+        Delete:
+          basePath: ${logDir}
+          IfLastModified:
+            age: 5d
```

Comment:
Just picked what I thought was a reasonable day. Not sure yet if these logs are written outside of the container. Don't need them to be for what I am planning. If they stay in the container then they are deleted on a pod restart. If a pod does not restart for 5+ days figured then would be a good time to clean up.